datashare-app/src/main/java/org/icij/datashare/mode/CommonMode.java

```diff
@@ -259,8 +291,8 @@ Routes addExtensionConfiguration(Routes routes) {
         return routes;
     }
 
-    protected boolean hasRedisProperty() {
-        return propertiesProvider.getProperties().contains(QueueType.REDIS.name());
+    protected boolean hasProperty(QueueType queueType) {
+        return propertiesProvider.getProperties().contains(queueType.name());
```

Comment:
I'm not sure about this. Does it check that any property in the property provider contains the queue type? Shouldn't we check the `queueType` property instead?

Reply:
Or `batchQueueType`.

Reply:
yes it was any property because there are `queueType` and `batchQueueType`. This is inherited  from all changes but as we talked about this yesterday with MV it will have to be harmonized. So the idea was : if there is AMQP anywhere we instantiate AMQP interlocutor, and if there is redis anywhere we do a RedisClient also;

Reply:
Got it, thanks!